pkg/antctl/antctl.go

```diff
@@ -531,6 +535,30 @@ var CommandList = &commandList{
 			supportController: true,
 			commandGroup:      get,
 		},
+		{
+			cobraCommand:      clusterset.Command,
+			supportAgent:      true,
+			supportController: true,
```

Comment:
I think we only support to run it out-of-cluster, right? if so, both of them should be `false`.